src/shared/components/query/QueryStore.ts

```diff
@@ -234,7 +234,7 @@ export class QueryStore
 	@observable private _maxTreeDepth:number = (window as any).maxTreeDepth;
 	@computed get maxTreeDepth()
 	{
-		return this.forDownloadTab ? 1 : this._maxTreeDepth;
+		return (this.forDownloadTab && this._maxTreeDepth > 0) ? 1 : this._maxTreeDepth;

```

Comment:
hi, not sure if i understand that logic here.  so, basically, the download tab can only ever have a depth of 0 or 1?  what's the reason for that?